notifier/tasks.py

```diff
@@ -104,6 +146,67 @@ def _time_slice(minutes, now=None):
     dt_start = dt_end - timedelta(minutes=minutes)
     return (dt_start, dt_end)
 
+
+@celery.task(max_retries=settings.DAILY_TASK_MAX_RETRIES, default_retry_delay=settings.DAILY_TASK_RETRY_DELAY)
+def do_forums_digests_flagged():
+    """
+    Generate and batch send digest emails for each thread with flagged posts.
+    """
+    def get_course_threads():
+        """
+        Parse threads from comments service API and group by course_id.
+
+        Returns:
+            dict: key=course_id, value=list of thread URLs with flagged posts
+        """
+        output = {}
+        for thread_url in get_flagged_threads():
+            match = re.search('^https?:\/\/\S+\/courses\/((?:[^\/]+\/){3})\S*', thread_url)
```

Comment:
Will this break for SplitMongo courses?

I can't remember what the API payload looks like; could you include a sample response here?

It _looks_ like the payload is just an array of URLs, is that correct?